elastic/kibana
# [Alert details page][Custom threshold] Add group by and tags fields to the alert details page summary

Closes #155698
Closes #174236

## Summary

This PR adds group and tag information to the alert details page summary fields for the custom threshold rule:

![image](https://github.com/elastic/kibana/assets/12370520/d93e32bd-99ae-414f-87ed-5ca5884acfa6)


## 🧪 How to test
- Enable alert details page config
    ```
    xpack.observability.unsafe.alertDetails.observability.enabled: true
    ```
- Create a custom threshold rule with a group by and add more than 3 rule tags
- After the alert is fired, go to the alert details page (from alert flyout or menu action in the alert table); you should see the group by and tags information there.
